### PR TITLE
doc: Improve uninstall

### PR DESCRIPTION
**Pull request by rtfpessoa**

### CODE REVIEW

File: docs/extra/uninstall.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+kubectl -n codacy get pod &
+kubectl -n codacy get pvc &
+kubectl -n codacy get pv  &
+kubectl -n codacy get job &
 ```
 
 Note that the deletion of `pvc`s in the above command has to run in the background
 due to a cyclic dependency in one of the components. If you are unsure of these
-commands' effect please run the each of the `bash` subcommands and validate their output, viz.
+commands' effect please run the each of the `bash` subcommands and validate their output.
```
Comment (pauloribeiro-codacy) [inline docs/extra/uninstall.md]: Suggestion and fix typo:

"If you are unsure of the effects of these commands please run each of the `bash` subcommands and validate their output."

Comment (rtfpessoa) [inline docs/extra/uninstall.md]:
```suggestion
commands please run each of the `bash` subcommands and validate their output.
```

File: docs/extra/uninstall.md [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+sleep 5
+kubectl -n codacy delete pod $(kubectl -n codacy get pod -o jsonpath='{.items[*].metadata.name}') --force --grace-period=0
+kubectl -n codacy get pod &
+kubectl -n codacy get pvc &
+kubectl -n codacy get pv  &
+kubectl -n codacy get job &
 ```
 
 Note that the deletion of `pvc`s in the above command has to run in the background
 due to a cyclic dependency in one of the components. If you are unsure of these
```
Comment (rtfpessoa) [inline docs/extra/uninstall.md]:
```suggestion
due to a cyclic dependency in one of the components. If you are unsure of the
effects of these commands please run each of the `bash` subcommands and validate their output.
```

Comment (rtfpessoa) [reply docs/extra/uninstall.md]: Like this @pauloribeiro-codacy ?

Comment (rtfpessoa) [inline docs/extra/uninstall.md]:
```suggestion
due to a cyclic dependency in one of the components. If you are unsure of the effects of these
```

File: docs/extra/uninstall.md (hunk 1 above)
Comment (rtfpessoa) [inline docs/extra/uninstall.md]:
```suggestion
commands please run each of the bash subcommands and validate their output.
```

**Pull request merged**